### DISCUSSION TITLE
The version 0.5.0 doesn't have the prefix `v`. Is it an intentional change?

### DISCUSSION THREAD

**Discussion by suzuki-shunsuke** [astral-sh/ruff#12078] [Q&A] [answered]
https://github.com/astral-sh/ruff/releases/tag/0.5.0

I expected the version was `v0.5.0`, but actually it was `0.5.0`.
I know 0.5.0 has some breaking changes, but this change isn't noted in the release note.
Is this an intentional change? Or just a miss?

**Comment by zanieb** [comment, marked as answer]
Yes this was an intentional change with our new release pipeline. Sorry it wasn't documented!

**Reply by zanieb** [reply]
This changed in https://github.com/astral-sh/ruff/pull/9559

**Reply by suzuki-shunsuke** [reply]
Thank you for your quick answer! 👍

**Reply by charliermarsh** [reply]
Out of curiosity: where are you relying on it? I'm just trying to understand how / which consumers might be affected.

**Reply by suzuki-shunsuke** [reply]
I'm a creator of [aqua](https://aquaproj.github.io/), which is a CLI version manager.
aqua has [an official registry](https://github.com/aquaproj/aqua-registry) and [it supports ruff](https://github.com/aquaproj/aqua-registry/blob/main/pkgs/astral-sh/ruff/registry.yaml).
When a tool which the official registry supports releases a new version, a pull request is created to test if a new version can be installed correctly.
When ruff 0.5.0 was released, we found it's breaking changes because CI failed.

- https://github.com/aquaproj/aqua-registry/pull/24368

We have already fixed the official registry to support ruff 0.5.0.

- https://github.com/aquaproj/aqua-registry/pull/24373

When I fixed the official registry, I found the version wasn't `v0.5.0` but `0.5.0`.

**Reply by Peiffap** [reply]
FYI, the pypi version badge in `README.md` still shows `v0.5.0`.

**Reply by charliermarsh** [reply]
I think that’s normal — that’s not based on the Git tag.
